### PR TITLE
Fix footer container.  #1180

### PR DESCRIPTION
**Pull request by JPOak**
Change:

- Issue #1180 - Update and fix footer container.

Note: I did not add a background color to ".footer" . I felt you should leave to the dev to decide whether to leave transparent or what have you. The structure is the important thing.

### CODE REVIEW

File: src/assets/scss/modules/_footer.scss [hunk 1]
```diff
@@ -1,11 +1,10 @@
 .footer-container {
```
Comment (colin-marshall) [inline src/assets/scss/modules/_footer.scss]: @JPOak do you think we should add a `.footer` selector here and just make the `background-color` transparent or white by default? The background color is the whole reason we were opting for option 2 after all.

**Comment by JPOak** [comment]
@colin-marshall I'll let you make the call on that. I'm happy to make the change. For me, it's just having the structural option there. It would not have to be as opinionated for me.

**Comment by colin-marshall** [comment]
@olefredrik what do you think?

**Comment by JPOak** [comment]
@colin-marshall This PR has been sitting here for awhile. Should I just withdraw it? Or change it? Just let me know. I like to clean up after myself : ) .

**Comment by colin-marshall** [comment]
Sorry @JPOak, was hoping for more feedback on what I proposed. Let's just get it merged for now.

**Pull request merged**